### DISCUSSION TITLE
Go Pro Time Sync 3.5mm Jam vs. QR Code

### DISCUSSION THREAD

**Discussion by chaudhuri-ab** [gopro/labs#448] [Q&A]
Just trying to understand what happens behind the scenes a bit with the Go Pro when you jam sync using the QR control vs. via 3.5 mm audio cable using LTC.

What I have seen using the QR code is that video recorded uses the local time that was jam synced via the QR code, which makes sense because the QR code from the app seems to only have a precise time encoded into it.

I have not yet had a chance to try the jam sync via 3.5mm, but when doing it that way does it impact the local time that the Go Pro keeps or leaves the local time alone and just updates time code data to start with the time code sent to it via the TC generator as there is only a frame number in the 80 bits of LTC?

Does the go pro use any other data other than just the Hour, Min, Sec, and Frame (i.e. it will ignore things like drop frame bit) when jam syncing via 3.5 mm?

<img width="933" alt="image" src="https://github.com/gopro/labs/assets/24656014/21c99a57-3720-45f5-bae7-7c87edf85079">

**Comment by dnewman-gpsw** [comment]
There is only one clock, for which timecode is determined.  If you LTC jam is sets the internal clock (Hour, minute and second, sub-second only.)
